frappe/public/js/frappe/list/base_list.js

```diff
@@ -345,6 +345,7 @@ frappe.views.BaseList = class BaseList {
 		const args = this.get_args();
 		return {
 			method: this.method,
+			type: 'GET',
```

Comment:
'POST' request is intentional here, because url parameters in a 'GET' request are limited to 4096 characters